src/main/kotlin/no/nav/syfo/client/isdialogmelding/IsdialogmeldingClient.kt

```diff
@@ -63,7 +64,12 @@ class IsdialogmeldingClient(
                     "Unable to determine fastlege, or lacking appropiate 'partnerinformasjon'-data",
                 )
                 COUNT_METRIKK_FOLLOWUP_LPS_DELT_MED_FASTLEGE_FALSE.increment()
-                throw GpNotFoundException()
+                if (isDev) {
+                    // There is currently no working test environment for sending plans to GP
+                    true
+                } else {
+                    throw GpNotFoundException()
+                }
```

Comment:
Min første tanke var at det kanskje er ryddigere å ha en toggle for dette. Men hvordan ønsker vi egentlig at det skal fungere? Er det slik at vi a) aldri ønsker å sende til fastlege i dev, eller b) at vi ønsker å forsøke å sende, men ikke gi feilmelding når det feiler?

Reply:
Det er vel enkelte testbrukere som har fastlege registrert, i tilfelle vi skulle ønske å teste det?

Reply:
Kan godt endre til toggle om du vil. Problemet i følge geir er at isdialogmelding sjekker både om fastlege finnes (det funker tror jeg), og om fastlegekontoret er tilnyttet emottak. Emottak-greiene funker ikke i test, finnes ikke noe testmiljø-oppsett for det. Så får vi samme type exception uavhengig av hvilke av disse to feilene som skjer.

Reply:
Geir sier han kan grave frem noen test-fnr som vil funke, dersom det hjelper oss. Men usikker på om det vil være helt optimalt, spesielt mtp det å teste om integrasjon funker for egen organisasjon (ikke-delegert)

Reply:
Så derfor tenkte jeg kanskje å bare droppe sending dersom det er dev, og bare si at det gikk fint.

Reply:
Hvis vi implementerer det som en toggle, så kan vi jo skru på togglen for en liten periode hvis vi har behov for å teste sending til fastlege.

Reply:
Yes :) Fikser